resources/test_data/sqlite_testrunner.tables

```diff
@@ -1,5 +1,6 @@
 resources/test_data/tbl/sqlite/mixed_types_100.tbl mixed
```

Comment:
(Random Line) @Felerius 

> When using this playground.cpp I don't find one. What am I missing here?

You are right. Since this is a nested subquery it will not have the parameter of the outer query in its arguments, and rightly so. We need to recursively craw subquery-LQPs....

Reply:
Done